oauth2/hook.go

```diff
@@ -23,6 +23,8 @@ type AccessRequestHook func(ctx context.Context, requester fosite.AccessRequeste
 type RefreshTokenHookRequest struct {
 	// Subject is the identifier of the authenticated end-user.
 	Subject string `json:"subject"`
+	// IDTokenExtra is arbitrary data set by the session.
+	IDTokenExtra map[string]interface{} `json:"id_token_extra"`
```

Comment:
Alternatively we could also include the full session, which would send all the context we have instead of just ID Token claims. WDYT?

```suggestion
	Session ThisNeedsToBeASwaggerAndJSONCompatibleRepresentationOfTheSessionObject `json:"session"`
```

Reply:
That sounds great - it would increase the flexibility for the webhook integrations. I'll make the change.